### DISCUSSION TITLE
Add User Registration Feature

### DISCUSSION THREAD

**Discussion by testusermayank** [himanshuchandola-zz/BRAINJET-QUIZ-PORTAL#92] [Q&A] [answered]
Add functionality of user registration and profile feature also leaderboard to compete with others

**Comment by himanshuchandola-zz** [comment, marked as answer]
Due to the lack of time atm, we are not actively working on the project. You may fork this repo and add that functionality. Open PR; after doing so, we will review and test it and merge it with our main repo.
